src/main/io/beeper.c

```diff
@@ -227,7 +231,8 @@ static const beeperTableEntry_t beeperTable[] = {
     { BEEPER_ENTRY(BEEPER_CAM_CONNECTION_OPEN,   20, beep_camOpenBeep,     "CAM_CONNECTION_OPEN") },
     { BEEPER_ENTRY(BEEPER_CAM_CONNECTION_CLOSE,  21, beep_camCloseBeep,    "CAM_CONNECTION_CLOSE") },
     { BEEPER_ENTRY(BEEPER_RC_SMOOTHING_INIT_FAIL,22, beep_rcSmoothingInitFail, "RC_SMOOTHING_INIT_FAIL") },
-    { BEEPER_ENTRY(BEEPER_ALL,                   23, NULL,                 "ALL") },
+    { BEEPER_ENTRY(BEEPER_ARMING_GPS_NO_FIX,     23, beep_armingGpsNoFix,  "ARMING_GPS_NO_FIX") },
```

Comment:
This should probably be moved up to be close to `BEEPER_ARMING_GPS_FIX`.

Reply:
Please, see conversation above with @McGiverGim :-)

Reply:
That is how it was at the beginning, but we are in doubt if the order matters or not :(